### PR TITLE
[ODS-6313] Scorecard supply-chain security Workflow Added

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: .github/workflows/Scorecard supply-chain security.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Originally sourced from GitHub with implicit lack of license
+
+name: Scorecard supply-chain security
+on:
+  pull_request:
```
Comment (simpat-jesus) [inline .github/workflows/Scorecard supply-chain security.yml:5]: Replace with a scheduled run

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Open a ticket to fix https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/security/code-scanning?query=pr%3A982+is%3Aopen

**Comment by semalaiappan** [comment]
> Open a ticket to fix https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/security/code-scanning?query=pr%3A982+is%3Aopen

New ticket created https://tracker.ed-fi.org/browse/ODS-6328

File: .github/workflows/Scorecard supply-chain security.yml (hunk 1 above)
Comment (semalaiappan) [reply .github/workflows/Scorecard supply-chain security.yml]: Fixed it . Ready for Review

**Review by vimayya — APPROVED** [review]

**Pull request merged**